azure/v14.0.1/README.md

```diff
@@ -0,0 +1,20 @@
+# :zap: Giant Swarm Release v14.0.1 for Azure :zap:
+
+This releases increases the Azure Events Termination timeout from 5 to 15 minutes for better upgrade experience while workloads are moved to new nodes
```

Comment:
```suggestion
This releases increases the Azure Events Termination timeout from 5 to 15 minutes for better upgrade experience while workloads are moved to new nodes.
```

Reply:
Does it make sense to mention the fact we updated the azure-scheduled-events app to make upgrade smoother?

Reply:
It is sort of in the applied changes